content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -54,17 +53,18 @@ kubectl create secret generic db-user-pass \
   --from-file=password=./password.txt
 ```
 
-You do not need to escape special characters in passwords from files
-(`--from-file`).
+You do not need to escape special characters in passwords from files that you
```

Comment:
Hi @shannonxtreme . The changes look good. Thanks.
nit:  What do you think about possibly rephrasing line 56 (the important bit is: special characters in the password string do not need to be escaped). Optionally, add the sentence as a note.
Something like:
When including a password from a file, you do not need to escape [the] special characters in the password string.

Reply:
This stumped me more than it should have, lol. I reworded to "You do not need to escape special characters in password strings that you include in a file." Let me know what you think.

Reply:
Sure. The original text was clumsy. Thanks!